request/graphql/parser.go

```diff
@@ -103,7 +103,10 @@ func (p *parser) Parse(ast *ast.Document) (*request.Request, []error) {
 	return query, nil
 }
 
-func (p *parser) ParseSDL(ctx context.Context, schemaString string) ([]client.CollectionDescription, error) {
+func (p *parser) ParseSDL(ctx context.Context, schemaString string) (
+	[]client.CollectionDescription,
+	error,
+) {
```

Comment:
question: Why this change?

Reply:
I changed this method's signature at some point but then changed it back. The method was too long to fit in one line. I can revert this change